README.md

```diff
@@ -1,3 +1,21 @@
 # portfolio-project
 
-## Biult With:
+> This is a Portfolio project to show case my tech skills. The development approach is mobile first.
+
+## Built With:
+
+- HTML
+- CSS
+- ES6lint
+
+## Using this Repository
+
+> Make us of the classic git clone concept to begin with. Ensure that you open the package.json file to see the dependencies that you install.
+
+## Author
+
+### Robertsn Akpan
+
+- GitHub: [@bobb-rob](https://github.com/bobb-Rob)
+- Twitter: [@\_\_Robertson](https://twitter.com/___Robertson)
+- LinkedIn: https://www.linkedin.com/in/robertson-arthur-6895a0123/
```

Comment:
In order to have a professional repo, it is better if we don't exclude sections from README.md file.

- [ ] Kindly, add back the following sections:
`Getting Started`
`Prerequisites` - what the user needs on his machine to reproduce your project
`Setup` - help with guided steps a non-professional user to clone your repo on his machine
`Install` - explain how  a non-professional user can run your project on his device
`Contributing` - just leave like it is in the template
`Show your support` - leave it as it is in the template
`Acknowledgments` - you can mention Microverse here and FIGMA template you use
`License` - leave it as it is in the template, but create in the root folder the `MIT.md` file edited appropriately; 

README Template can be found here: [Template](https://github.com/microverseinc/readme-template);

**After this, definitely, your repo will look professional** :1st_place_medal: